modules/ingestor/src/service/cve/loader.rs

```diff
@@ -51,9 +55,18 @@ impl<'g> CveLoader<'g> {
             )));
         }
 
+        let information = AdvisoryInformation {
+            title: cve.containers.cna.title,
+            published: match cve.cve_metadata {
+                CveMetadata::Published(ref m) => m.date_published,
+                CveMetadata::Rejected(ref m) => m.date_published,
```

Comment:
Rejected == Published?

Reply:
Dunno. It's a weird structure. It was just my best guess. Gotta better one?

Reply:
Do we need more fields in the Information?

Reply:
Or in the entity?

Reply:
I don't know what those CVE fields mean.